src/components/Feilmeldingspanel/Feilmeldingspanel.tsx

```diff
@@ -0,0 +1,23 @@
+import Panel from 'nav-frontend-paneler';
+import { Feiloppsummering, FeiloppsummeringFeil } from 'nav-frontend-skjema';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface FeilmeldingspanelProps {
+  feilmeldinger: Array<FeiloppsummeringFeil>;
+}
+
+const Feilmeldingspanel = (props: FeilmeldingspanelProps) => {
+  const { t } = useTranslation();
+  if (props.feilmeldinger && props.feilmeldinger.length > 0) {
+    return (
+      <Panel>
+        <Feiloppsummering tittel={t('FEILMELDINGSPANEL')} feil={props.feilmeldinger} />
```

Comment:
Mulig teksten burde sendes inn som en prop? Nå binder du den 'FEILMELDINGSPANEL' statusen. Og hvor kommer den fra?

Reply:
Pushet ny commit for denne